lib/suspenders/generators/app_generator.rb

```diff
@@ -239,7 +239,13 @@ def remove_routes_comment_lines
 
     def outro
       say 'Congratulations! You just pulled our suspenders.'
-      say "Remember to run 'rails generate airbrake' with your API key."
+
+      if options[:heroku]
+        say "Run 'bundle exec honeybadger heroku install' with your API key
+        unless you're using the Heroku Honeybadger add-on."
```

Comment:
I don't know how this message will appear, but judging by the indentation here, it may look like:

```
Run 'bundle exec honeybadger heroku install' with your API key
        unless you're using the Heroku Honeybadger add-on.
```

What do you think about using a heredoc to ensure that this message is indented correctly?